tests/charts/api_tests.py

```diff
@@ -564,12 +564,13 @@ def test_update_chart_new_owner(self):
         gamma = self.get_user("gamma")
         admin = self.get_user("admin")
         chart_id = self.insert_chart("title", [gamma.id], 1).id
-        chart_data = {"slice_name": "title1_changed"}
+        chart_data = {"slice_name": "title1_changed", "owners": [admin.id]}
```

Comment:
Would be nice to add a `gamma` user test also. Seems to be missing the same kind of test for reports and datasets